lms/djangoapps/teams/static/teams/js/spec/views/manage_spec.js

```diff
@@ -24,29 +24,26 @@ define([
         });
 
         it('can render itself', function() {
-            expect(_.strip(view.$('.download-team-csv').text())).toEqual('Download Memberships');
-            expect(_.strip(view.$('.upload-team-csv').text())).toEqual('Upload Memberships');
+            expect(_.strip(view.$('#download-team-csv').text())).toEqual('Download Memberships');
```

Comment:
Mostly for my education: I recall using classes instead of id's as a recommended practice. Has this recommendation changed? I.e: why are we using id selectors here?

Reply:
Is that an edX practice? For targeting a single element on a page I've always seen the guidance to use ID since it semantically refers to a unique element.

Reply:
Not edX I heard this sometime ago and just wanted to check. I don't recall seeing a specifically "recommended" way.

Reply:
Good response to a similar question: https://stackoverflow.com/a/1230650/8507437